setup.py

```diff
@@ -98,7 +98,8 @@
             'test = tracim.command:TestTracimCommand',
             'user_create = tracim.command.user:CreateUserCommand',
             'user_update = tracim.command.user:UpdateUserCommand',
-            'db_init = tracim.command.initializedb:InitializeDBCommand',
+            'db_init = tracim.command.database:InitializeDBCommand',
+            'db_delete = tracim.command.database:DeleteDBCommand',
```

Comment:
this kind of command should use a mandatory --force option in order not to drop database by error. By the way, this command should also remove files in depot folder, doesn't it?